doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,168 @@
-
-#### 2. Remove existing generated code
-
-Open the `Azure.Template.Generated.sln` file from the `sdk\<your-service-name>\Azure.Template.Generated` directory.
-
-You'll notice that the `Azure.Template.Generated` project already has generated code in it!  You're welcome to explore what's there -- it's been generated from the swagger file in the `sdk/template-dpg/Azure.Template.Generated/src/swagger` folder. For this tutorial, however, we'll delete these generated files so we can start clean.
-
-- Delete the Generated folder in the `Azure.Template.Generated` project.
-- Delete the `swagger` folder in the `Azure.Template.Generated` project.
-
-#### 3. Rename the solution and projects
-
-In `Azure.Template.Generated` solution, you will notice that it contains `Azure.Template.Generated.csproj` and `Azure.Template.Generated.Tests.csproj`. You will need to rename the project and solution based on your `Azure.<group>.<service>` library. For now, you can remove the `Azure.Template.Generated.Tests` test project reference from the solution and add back when the generated code is ready.
-
-In Visual Studio:
-
-- Rename the `Azure.Template.Generated` project to `Azure.<group>.<service>`.
-- Rename the `Azure.Template.Generated` solution to `Azure.<group>.<service>`.
-- Remove the `Azure.Template.Generated.Tests` project from the solution.
-
-**Note**: `Azure.<group>.<service>` is the Azure SDK package name. The package name/namespace will be something the architects will help standardize according to the guidelines.
-
-**Learn more:** see the [.NET Guidelines on Namespace Naming](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespace-naming) for more information on naming Azure SDK packages. This name diverges from approved Azure SDK namespaces, since it is intended to be used just for this tutorial.
-
-#### 4. Update the Assembly Info
+e.g.
+```
+pwsh /home/azure-sdk-for-net/eng/scripts/automation/Invoke-DataPlaneGenerateSDKPackage.ps1 -service sample -namespace Azure.Template.Sample -sdkPath /home/azure-sdk-for-net -inputfiles https://github.com/Azure/azure-rest-api-specs/blob/73a0fa453a93bdbe8885f87b9e4e9fef4f0452d0/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json -securityScope https://sample/.default
+```
 
-You'll notice that the `Azure.<group>.<service>` project has `properties/AssemblyInfo.cs` file in it. Replace `Microsoft.Test` namespace with the correct resource provider namespace for your service.
+**Note**: 
+- Use one of the following pre-approved namespace groups (https://azure.github.io/azure-sdk/registered_namespaces.html): Azure.AI, Azure.Analytics, Azure.Communication, Azure.Data, Azure.DigitalTwins, Azure.IoT, Azure.Learn, Azure.Media, Azure.Management, Azure.Messaging, Azure.ResourceManager, Azure.Search, Azure.Security, Azure.Storage, Azure.Template, Azure.Identity, Microsoft.Extensions.Azure
```

Comment:
Do we need the namespaces to be listed here? How will we keep it in sync with what's at https://azure.github.io/azure-sdk/registered_namespaces.html?

Reply:
In the webside, it just list the registered namespaces, not the approved groups. Currently we have no webside to list approved groups, so I list in the doc. If we have website for groups, I will update the doc with the link,

Reply:
The groups are listed here: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespaces-approved-list

Reply:
add the approved groups websit link in the doc